src/Codeception/Configuration.php

```diff
@@ -500,6 +500,126 @@ public static function projectDir()
     }
 
     /**
+     * Returns path to a given directory relative to $projDir.
+     * @param string $path
+     * @param string $projDir
+     * @param string $dirSep
+     * @return string
+     */
+    public static function getRelativeDir($path, $projDir, $dirSep = DIRECTORY_SEPARATOR)
+    {
+        // ensure $projDir ends with a trailing $dirSep
+        $projDir = preg_replace('/'.preg_quote($dirSep, '/').'*$/', $dirSep, $projDir);
```

Comment:
No an issue, but mu preferred way is $projDir = rtrim($projDir, $dirSep) . $dirSep;